ecosystem/indexer/migrations/2022-04-07-001831_create_processor_status/up.sql

```diff
@@ -0,0 +1,14 @@
+-- Your SQL goes here
+
+
+CREATE TABLE processor_statuses
```

Comment:
makes debugging much easier than logs or counters: those are aggregates, these are absolute specifics, and because they're in the DB, it makes recovery after restarts, crashes, bugfixes, etc possible :-)

Every version's processing state is saved, per processor, in this table: on start, and updated on completion. If your processor has any problems and can't parse a version correctly, and you make some changes to said processor, the indexer will automatically retry all previously failed versions on start :-)